fractions/wildfly/elytron/src/main/java/org/wildfly/swarm/elytron/ElytronFraction.java

```diff
@@ -239,35 +249,28 @@ public ElytronFraction applyDefaults() {
             }});
             auth.mechanismConfiguration(new HashMap() {{
                 put(MECHANISM_NAME, "DIGEST-MD5");
-                put("mechanism-realm-configuration", new ArrayList() {{
+                put(MECHANISM_REALM_CONFIGURATIONS, new ArrayList() {{
                     add(new HashMap() {{
                         put(REALM_NAME, APPLICATION_REALM);
                     }});
                 }});
             }});
         });
 
-        providerSaslServerFactory(GLOBAL);
+        configurableSaslServerFactory("configured", (configurable) -> {
+            configurable.saslServerFactory(ELYTRON);
+            configurable.property("wildfly.sasl.local-user.default-user", "$local");
+        });
 
         mechanismProviderFilteringSaslServerFactory(ELYTRON, (filtering) -> {
             filtering.saslServerFactory(GLOBAL);
             filtering.filter(new HashMap() {{
-                put("provider-name", ELYTRON);
+                put("provider-name", "WildFlyElytron");
             }});
 
         });
 
-        configurableSaslServerFactory("configured", (configurable) -> {
-            configurable.saslServerFactory(ELYTRON);
-            configurable.filter(new HashMap() {{
-                put("pattern-filter", "JBOSS-LOCAL-USER");
-            }});
-            configurable.filter(new HashMap() {{
-                put("pattern-filter", "DIGEST-MD5");
-            }});
-            configurable.property("wildfly.sasl.local-user.default-user", "$local");
-        });
-
+        providerSaslServerFactory(GLOBAL);
```

Comment:
What we had here was quite different than the default Elytron config in WildFly 11. I _guess_ that was because this file was created a while before WildFly 11 was released -- I think Elytron was changing quite heavily for a long time.

However, if there was some _other_ reason why this was so different, I'd like to know.